addon/templates/components/power-select-multiple/trigger.hbs

```diff
@@ -17,7 +17,7 @@
     </li>
   {{else}}
     {{#if (and placeholder (not searchEnabled))}}
-      <span class="ember-power-select-placeholder">{{placeholder}}</span>
```

Comment:
One last comment. The way the placeholder component works in the single component is that you render it, and it already check if its `placeholder` property is there to decide wether it renders anything or not.

In think that you simplify this to

```hbs
{{#if (not searchEnabled)}}
  {{component placeholderComponent placeholder=placeholder}}
{{/if}}
```